index.js

```diff
@@ -88,6 +88,26 @@ export async function load(options = {
   } else {
     navigator.credentials = polyfill.credentials;
   }
+
+  // set up proxy for `navigator.credentials`, to ensure subsequent changes
+  // to it do not prevent the polyfill from running
+  const navCredentialsProxy = new Proxy(navigator.credentials, {
+    set(obj, prop, value) {
+      if(prop in polyfill.credentials._nativeCredentialsContainer) {
+        // replace underlying function, keeping credentials polyfill intact
+        polyfill.credentials._nativeCredentialsContainer[prop] = value;
```

Comment:
We might find that we need to use `defineProperty` here as well... but we can wait to hit that problem (if it exists out there), I guess.